style.css

```diff
@@ -547,3 +547,494 @@ hr {
   border-radius: 10px;
   background-color: black;
 }
+
+.title hr {
+  width: 100%;
+  height: 1px;
+}
+
+/* desktop-version */
+
+@media only screen and (min-width: 760px) {
```

Comment:
- [ ] The general layout of the design but is not responsive. as I change the width of the screen. the elements stay a fixed size. It's important that you make the design responsive so that your design works on different sizes of desktop and mobile devices.
## Your Mobile Design ( please notice the whitespace on the right)
![image](https://user-images.githubusercontent.com/18730106/145454323-d0cc2708-3613-4c5f-af9f-d4f5bf965d54.png)
your design should street the whole page and stay responsive
- [ ] Look at the `header` area you can see the text on the right is getting cut off this shouldn't be the case. please fix this issue.
# Your Header desktop ( notice the texts on the right )
![image](https://user-images.githubusercontent.com/18730106/145455030-39345113-52d0-47bf-a658-de1d37832596.png)